ticket.go

```diff
@@ -171,7 +171,20 @@ func (c *Conn) encryptTicket(state []byte) ([]byte, error) {
 	return encrypted, nil
 }
 
+// [UTLS] changed to use exported DecryptTicketWith func below
 func (c *Conn) decryptTicket(encrypted []byte) (plaintext []byte, usedOldKey bool) {
+	tks := ticketKeys(c.config.ticketKeys()).ToPublic()
+	return DecryptTicketWith(encrypted, tks)
+}
+
+// DecryptTicketWith decrypts an encrypted session ticket
+// using a TicketKeys (ie []TicketKey) struct
+//
+// usedOldKey will be true if the key used for decryption is
```

Comment:
"DecryptTicketWith returns the decrypted session ticket, and a boolean usedOldKey,
which will be true, if the key used for decryption was not the first in the []TicketKey slice."

I don't see where usedOldKey is set though.

Reply:
Yeah I thought that it was confusing the way they were using a named return there too:
https://github.com/refraction-networking/utls/blob/33a29038e742910d6ec82636748dfad5dd7f30ae/ticket.go#L213

I think maybe it was as a form of documentation?